### PR TITLE
Fix some minor typos

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [ ] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/supported-features-oracle.adoc [hunk 1]
```diff
@@ -119,8 +119,8 @@ The icons in this table indicate:
 | {salt}
 
 | Contact methods
-| {trad}: OSAD, RHNSD, SSH-push. {salt}: ZeroMQ, Salt-SSH
-| {trad}: OSAD, RHNSD, SSH-push. {salt}: ZeroMQ, Salt-SSH
+| {trad}: OSAD, RHNSD, SSH-push. {salt}: ZeroMQ, Salt-SSH
+| {trad}: OSAD, RHNSD, SSH-push. {salt}: ZeroMQ, Salt-SSH
```
Comment (Loquacity) [inline modules/client-configuration/pages/supported-features-oracle.adoc:123]: What's going on here?

**Review by Loquacity — APPROVED** [review]

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> What's going on here?

I removed some no-break space characters, AKA "{nbsp}":

            character:   (displayed as  ) (codepoint 160, #o240, #xa0)
              charset: unicode (Unicode (ISO10646))
code point in charset: 0xA0

They were in right after the initial "|" characters.  With Emacs, I can
see them; my Emacs displays them with dark red underline.

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
> Lana Brindley <notifications@github.com> writes:
> What's going on here?
> I removed some no-break space characters, AKA "{nbsp}": character:   (displayed as  ) (codepoint 160, #o240, #xa0) charset: unicode (Unicode (ISO10646)) code point in charset: 0xA0 They were in right after the initial "|" characters. With Emacs, I can see them; my Emacs displays them with dark red underline.
> […](#)
> -- Karl Eichwalder

Huh, interesting. You'd think that would've broken the tables :thinking:

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Huh, interesting. You'd think that would've broken the tables :thinking: 

It should ;)  But I did not check that.

-- 
Karl Eichwalder

**Review by jcayouette — APPROVED** [review]

**Pull request merged**